### PR TITLE
Add a fullflow feature for judgments

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: src/test/resources/features/JudgmentFullflow.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Feature: Judgment Full user journey
+  Scenario: Full judgment transfer workflow
+    Given A logged out judgment user
+    When the user navigates to TDR Start Page
+    And the user clicks on the Start now button
+    And the logged out user enters valid credentials
+    And the user clicks the continue button
+    Then the user should be on the homepage page
+    When the user clicks the continue button
```
Comment (TomJKing) [inline src/test/resources/features/JudgmentFullflow.feature:11]: Is the button name not `Start Transfer` for judgment users? Should probably update the step to reflect this.

File: src/test/resources/features/JudgmentFullflow.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#Feature will not work with chromedriver in headless mode due to following bug: https://bugs.chromium.org/p/chromedriver/issues/detail?id=2521&q=directory%20upload&colspec=ID%20Status%20Pri%20Owner%20Summary
+
+Feature: Judgment Full user journey
+  Scenario: Full judgment transfer workflow
+    Given A logged out judgment user
+    When the user navigates to TDR Start Page
+    And the user clicks on the Start now button
+    And the logged out user enters valid credentials
+    And the user clicks the continue button
```
Comment (TomJKing) [inline src/test/resources/features/JudgmentFullflow.feature:9]: Is the button name not `Start Transfer` for judgment users? Should probably update the step to reflect this.

File: src/test/resources/features/JudgmentFullflow.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    Then the user should be on the homepage page
+    When the user clicks the continue button
+    Then the user should be on a page with before-uploading and a consignmentId in the URL
+    When the user clicks on the Continue button
+    Then the user will be on a page with the title "Upload judgment"
+    When the user selects the file: testdocxfile.docx
+    And the user clicks the continue button
```
Comment (TomJKing) [inline src/test/resources/features/JudgmentFullflow.feature:16]: Is the button name not `Start upload` for judgment users? Should probably update the step to reflect this.

File: src/test/resources/features/JudgmentFullflow.feature [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    Then the user will be on a page with the title "Uploading judgment"
+    Then the user will be on a page with the title "Checking your upload"
+    Then the user will be on a page with the title "Results of checks"
+    When the user clicks the continue button
```
Comment (TomJKing) [inline src/test/resources/features/JudgmentFullflow.feature:20]: Is the button name not `Export` for judgment users? Should probably update the step to reflect this.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some minor comments. Thanks

File: src/test/resources/features/JudgmentFullflow.feature (hunk 2 above)
Comment (techncl) [reply src/test/resources/features/JudgmentFullflow.feature:9]: This one is "Sign in"; I'll change it to that.

**Review by TomJKing — DISMISSED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]
Looks god. Thanks

**Pull request merged**